src/multidim/box_macro.jl

```diff
@@ -0,0 +1,42 @@
+using ValidatedNumerics
+
+
+"""Macro to make a version of a multidimensional function that acts on
+an `IntervalBox` and returns an `IntervalBox`.
+Both the original n-argument function and the `IntervalBox` version are defined.
+
+Example:
+
+    @box f(x, y) = (x + y, x - y)
+
+    X = IntervalBox(1..1, 2..2)
+    f(X)
+
+(No significant error checking is performed!)
+"""
+
+macro box(ex)
```

Comment:
Could we rename @box` to `@intervalbox` or `@boxinterval`, or whatever name that points out that we are creating an *interval* box? At the moment we have `@biginterval`, which I like because it's pretty clear what it does.